src/main/scala/com/typesafe/sbt/pom/PomReaderPlugin.scala

```diff
@@ -0,0 +1,15 @@
+package com.typesafe.sbt.pom
+
+import sbt._
+import sbt.plugins.JvmPlugin
+
+/** Plugin definition for Maven POM reader. */
+object PomReaderPlugin extends AutoPlugin {
+  object autoImport extends SbtPomKeys {
+    @deprecated("This is an AutoPlugin; settings are automatically added. Using this value is no longer required", "1.1")
+    lazy val useMavenPom = projectSettings
+  }
+  override def requires = JvmPlugin
+  override def trigger = allRequirements
```

Comment:
I think this should actually not be "allRequirements", but we should make users explciitly enable it....

I'm on the fence.  What's your inclination here?